docs/user-guide.md

```diff
@@ -41,9 +41,14 @@ This guide provides detailed instructions on how to import and configure your Li
 
 ### Settings
 
-Liberty Tools for IntelliJ IDEA honours  the following external settings:
-- Maven home path set in the **Preferences > Build, Execution, Deployment > Build Tools > Maven** window when running Liberty dev mode on Maven projects.
-- Gradle JVM set in the **Preferences > Build, Execution, Deployment > Build Tools > Gradle** window when running Liberty dev mode on Gradle projects.
+Liberty Tools for IntelliJ IDEA honors the following settings:
+
+#### IntelliJ settings:
+- Maven home path set in the **Settings > Build, Execution, Deployment > Build Tools > Maven** window when running Liberty dev mode on Maven projects.
+- Gradle JVM set in the **Settings > Build, Execution, Deployment > Build Tools > Gradle** window when running Liberty dev mode on Gradle projects.
```

Comment:
```suggestion
- The Gradle JVM that is set in the **Settings > Build, Execution, Deployment > Build Tools > Gradle** window when you run Liberty dev mode on Gradle projects.
```

Reply:
@evie-lau I think we should mention the Gradle "Distribution" setting instead, which I believe is what LT hounours... but we should double check

Reply:
It does indeed honor the Distribution setting. 
But for the Gradle JVM setting just below it, I don't notice any difference when changing that.